src/integration/common/mediaPlayer.js

```diff
@@ -0,0 +1,13 @@
+export default () => {
+  describe('Media player', () => {
+    describe('A11y', () => {
+      it('I can read the media player title', () => {
+        const mediaPlayerEl = document.querySelector('iframe, amp-iframe');
+
+        expect(mediaPlayerEl).toBeInTheDocument();
+        expect(mediaPlayerEl.getAttribute('src')).toBeTruthy();
```

Comment:
I think this is supposed to be the `title` attribute, instead of `src` (the snapshot is `"https://polling.test.bbc.co.uk/ws/av-embeds/media/bbc_amharic_radio/liveradio/am/amp"` when it should be `Audio player`, `Media player` (or the equivalent translation).